tf_agents/environments/atari_wrappers.py

```diff
@@ -94,3 +94,19 @@ def _step(self, action):
   @property
   def game_over(self):
     return self._num_steps >= self._duration or self.gym.game_over
+
+
+class FireOnReset(gym.Wrapper):
+  """Start every episode with action 1 (FIRE) + another action (2).
+
+  This is required by some environments (e.g., Breakout) to actually start the
+  game.
+  """
+
+  def reset(self):
+    observation = self.env.reset()
+    action_meanings = self.env.unwrapped.get_action_meanings()
+    if action_meanings[1] == 'FIRE' and len(action_meanings) >= 3:
```

Comment:
can you add a comment about the hard-coded integesr (len >= 3) used here and env.step arguments?

Reply:
I'd love to, but I'm really not sure. I just reused the same logic as in [this code](https://github.com/openai/gym/blob/master/gym/wrappers/atari_preprocessing.py#L104), which does not explain why it does that. I assume that some games require the fire button followed by any other action except NOOP (which is action 0). I'm not sure which ones, though. Would you like me to write that?

Reply:
ok, no problem. you can leave out the comment since it comes from a the gym code

Reply:
Add a comment referencing the code you lifted this from?